local-proxy/src/server.ts

```diff
@@ -66,10 +66,20 @@ app.post('/invoke', json(), async (req, res) => {
   const { path, handler, args } = req.body;
   registry.register(requestId);
   try {
-    await transpilePromise;
-    // Make sure we use tmpDir as absolute path
-    const mod = require(pathJoin(tmpDir, path));
-    const fn = mod[handler];
+    let fn: (...args: any[]) => any;
+    if (path === '@binaris/shift-db' && (handler === 'getVersioned' || handler === 'poll')) {
```

Comment:
This is not nice hard-coding.

```suggestion
    if (path === '@binaris/shift-db') {
      const shiftDb = require(path);
      ret = shiftDb[handler];
      return ret[magicExportToBackendSymbol] && ret;
    }
```

Reply:
Going to do what @vogre suggested below.